dev/automated_tests/flutter_test/package_assets_test.dart

```diff
@@ -13,7 +13,6 @@ void main() {
         width: 54,
         height: 54,
         fit: BoxFit.none,
-        package: 'flutter_automated_tests',
```

Comment:
The framework doesn't ever request assets with a package name when loading assets from the current package. This only worked because the asset loading workaround in https://github.com/flutter/flutter/blob/master/packages/flutter_test/lib/src/_binding_io.dart#L34 was not fully accurate to normal behavior